optuna/integration/allennlp.py

```diff
@@ -21,6 +21,32 @@
     TrackerCallback = object
 
 
+def _dump_best_config(config_file: str, study: optuna.Study) -> Dict:
```

Comment:
Is thin function intended to be reused somewhere? I think it's trivial enough that we can inline it in `dump_best_config` otherwise.

Reply:
You're absolutely right.
Removed `_dump_best_config` in f441713.